src/html5/http.js

```diff
@@ -87,9 +87,12 @@ export class Html5HttpMiddleware extends Middleware {
         timeout,
         followRedirect
       } = request;
+      const kinveyUrlRegex = /kinvey\.com/gm;
 
-      // Add the X-Kinvey-Device-Information header
-      headers['X-Kinvey-Device-Information'] = deviceInformation(this.pkg);
+      if (kinveyUrlRegex.test(url)) {
+        // Add the X-Kinvey-Device-Information header
+        headers['X-Kinvey-Device-Information'] = deviceInformation(this.pkg);
```

Comment:
How is the setting of the `X-Kinvey-Device-Information` header related to the issue of calling success when an error is returned from the backend?

Reply:
I was getting errors where Google was reject the `GET` request because of the `X-Kinvey-Device-Information` header. It was not causing the issue but it was preventing me from testing. I went ahead and made the change here.